README.md

```diff
@@ -15,51 +15,96 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  result = l[0]
+  if len(l) == 0:
+    return None
+  for i in l:
+    if i > result:
+      result = i
+  return result
```

Comment:
Great job solving this while practicing loops! This is exactly what I was looking for. In the future, you can also solve this using built in `min` and `max` functions.